libraries/core-react/src/hooks/usePopper.ts

```diff
@@ -0,0 +1,53 @@
+import { usePopper as reactPopper } from 'react-popper'
+
+export const usePopper = (
+  anchorEl: HTMLElement,
+  popperEl: HTMLElement,
+  arrowRef?: HTMLElement | string,
+  placement?:
+    | 'auto'
+    | 'auto-start'
+    | 'auto-end'
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'right'
+    | 'right-start'
+    | 'right-end'
+    | 'left'
+    | 'left-start'
+    | 'left-end',
```

Comment:
Suggest making a seperate type for `Placements` and export this for use in the individual components instead of duplicating string literal.